config/php/docker-entrypoint.sh

```diff
@@ -62,6 +62,32 @@ function waitUntil() {
   done
 }
 
+function setPermissions()
```

Comment:
I'd suggest function name be more like `createApplicationDirs` or `createVarDirs` - setting permissions is just a way to achieve the effect, not the intention itself